website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -111,40 +111,139 @@ models:
 
 ## Materialized views
 
-The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html).
-Redshift-specific configuration includes the typical `dist`, `sort_type`, `sort`, and `backup`.
-For materialized views, there is also the `auto_refresh` setting, which allows Redshift to [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you.
-The remaining configuration follows the general [materialized view](/docs/build/materializations#Materialized-View) configuration.
-There are also some limitations that we hope to address in the next version.
+The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html)
+with the following configuration parameters:
+
+| Parameter                                 | Type         | Required | Default              | Change Monitoring Support |
+|-------------------------------------------|--------------|----------|----------------------|---------------------------|
+| `on_configuration_change`                 | `<string>`   | no       | `apply`              | n/a                       |
+| [`dist`](#using-sortkey-and-distkey)      | `<string>`   | no       | `even`               | drop/create               |
+| [`sort`](#using-sortkey-and-distkey)      | `[<string>]` | no       | `none`               | drop/create               |
+| [`sort_type`](#using-sortkey-and-distkey) | `<string>`   | no       | `auto` if no `sort`  | drop/create               |
+|                                           |              |          | `compound` if `sort` |                           |
```

Comment:
@mikealfare are you intending a [mostly blank row](https://docs-getdbt-com-git-content-materialized-viewsredshift-dbt-labs.vercel.app/reference/resource-configs/redshift-configs#materialized-views) for the `compound` if `sort` portion? Or were you wanting it to be on the same row as `sort_type` to appear immediate after `auto` if no `sort`?

<img width="763" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/b9009b8d-9f8d-47d6-b55f-6c396e1d1db2">

If the latter, you can add a `<br />` (or two) to separate those two different cases within the same cell or rewrite it slightly like this

| Parameter                                 | Type         | Required | Default              | Change Monitoring Support | 	
|-------------------------------------------|--------------|----------|----------------------|---------------------------|
| `on_configuration_change`                 | `<string>`   | no       | `apply`              | n/a                       |
| [`dist`](#using-sortkey-and-distkey)      | `<string>`   | no       | `even`               | drop/create               |
| [`sort`](#using-sortkey-and-distkey)      | `[<string>]` | no       | `none`               | drop/create               |
| [`sort_type`](#using-sortkey-and-distkey) | `<string>`   | no       | `auto` if not `sort` (`compound` otherwise)    | drop/create               |
| [`auto_refresh`](#auto-refresh)           | `<boolean>`  | no       | `false`              | alter                     |
| [`backup`](#backup)                       | `<string>`   | no       | `true`               | n/a                       |

Reply:
I was having a brain fart about that. Yes, that's what I wanted to do. I updated it.